filebeat/module/elasticsearch/audit/_meta/fields.yml

```diff
@@ -0,0 +1,40 @@
+- name: audit
+  type: group
+  description: >
+  fields:
+    - name: node_name
```

Comment:
Is this the Elasticsearch node name? If yes, this field should end up under `elasticsearch.node.name`. Like this we will be able to correlate it.

Reply:
I am investigating here a bit more, because we have several options and more fields can show up. I need to run different configurations to follow up on behaviour and best approach. I will keep in mind that this field could be moved.